surface/src/3rdparty/opennurbs/opennurbs_compress.cpp

```diff
@@ -536,7 +536,7 @@ bool ON_UncompressStream::End()
 
   const ON__UINT32 sizeof_out_buffer = (ON__UINT32)(sizeof(imp->m_zlib_out_buffer));
   void* out_buffer = imp->m_zlib_out_buffer;
-  int zrc = Z_OK;
+  int zrc;
```

Comment:
You can reduce the scope of this variable since you've already touched the 3rd party code